scalatest/src/main/scala/org/scalatest/tools/DashboardReporter.scala

```diff
@@ -853,7 +853,7 @@ private[scalatest] class DashboardReporter(directory: String,
       
       if (event.throwable.isDefined) {
         val throwable = event.throwable.get
-        val stackTrace = throwable.getStackTrace
+        val stackTrace = throwable.getStackTrace.map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.